src/AFQMC/Hamiltonians/HSPotential_Helpers.cpp

```diff
@@ -78,9 +78,8 @@ void count_over_cholvec(double cut,
       if (abs(*vik) > cut)
       {
         count[2 * (*cik)] += size_t(2); // Lik + 0
-#if defined(QMC_COMPLEX)
-        count[2 * (*cik) + 1] += size_t(2); // Lik - 0
-#endif
+        if constexpr (is_complex<T>::value)
```

Comment:
```suggestion
        if constexpr (is_complex_v<T>)
```
Requires the helper function I suggested adding in config.0.h